override_extension_draw.py

```diff
@@ -0,0 +1,522 @@
+# This is a separate library that overrides the extension_draw_item method from Blender extensions list display.
+# the original code is in the bl_extension_ui.py file in the Blender source code.
+# The override library can be placed in multiple addons, and the override should happen only once.
+# The override is done by replacing the original method with the new one, and backing up the original method.
+# The original method is then called from the new method, with the same arguments, but with the new code added.
+
+
+import json
+import os
+import bpy
+import bl_pkg.bl_extension_ui as exui
+from . import icons
+from bl_ui.space_userpref import (
+    USERPREF_PT_addons,
+    USERPREF_PT_extensions,
+    USERPREF_MT_extensions_active_repo,
+)
+from bpy.props import EnumProperty
+
+EXTENSIONS_API_URL = "https://www.blenderkit.com/api/v1/extensions/"
```

Comment:
Should be constructed from global_vars.SERVER not hardcoded so it can be used for testing with devel or staging.

Reply:
the is actually intentional, so that this  separate library can be used in other addons too.